wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,427 @@
+---
+tags:
+  usage
+  permissions
+---
+
+# Usage permissions
+
+## Opt-in
+
+If you are an artist and wish to have your stance regarding whether osu! users may make use of your work known publicly or privately, please feel free to reach out to either [the osu! support team via email](mailto:support@ppy.sh) or any member of the [osu!team](/wiki/People/osu!team) with as much information as possible regarding what work you may want to allow or disallow.
```

Comment:
the first clause before comma here is kind of hard to understand with how much it's trying to say... and also due to having the word "known" so far apart from the subject it's acting on ("your stance") i think. the "publicly or privately" part should be explained separately in a new sentence or something, I had to re-read this a few times to understand what was meant by that

so in general just try to simplify the sentence structure I guess. I think it's especially important for an article like this to be a clear and easy read.

Reply:
"any member of the osu! team" – no
"support@ppy.sh" – no

please change this to require explicitly contacting `music@ppy.sh`, no other avenues will be accepted.

```suggestion
If you are an artist and wish to add or update your entry on this listing, please feel free to reach out to our [music licencing team](mailto:music@ppy.sh).
```